Intersect.Client/Entities/Player.cs

```diff
@@ -1150,7 +1126,7 @@ public bool TryTarget()
                                 if (en.Value.CurrentMap == mapId &&
                                     en.Value.X == x &&
                                     en.Value.Y == y &&
-                                    (!en.Value.IsStealthed() || Globals.Me.IsInMyParty(en.Value)))
+                                    (!en.Value.IsStealthed() || en.Value is Player player && Globals.Me.IsInMyParty(player)))
```

Comment:
This was fine, doesn't need changing.

Reply:
Definitely does, en is a generic entity here.
IsInMyParty no longer accepts those.